### DISCUSSION TITLE
libuv support

### DISCUSSION THREAD

**Discussion by CaptainOnly** [dbus-cxx/dbus-cxx#117] [Ideas]
Has anyone worked on libuv support? It looks like implementing a dispatcher like in dbus-cxx-qt or dbus-cxx-glib would do the trick. Is that the right approach?

**Comment by rm5248** [comment]
I do not believe that anybody has done anything with libuv.

If you want to support it, all you need to do is to extend the `Dispatcher` class and create an implementation for it.  Looking at the implementations that already exist for Qt and GLib is definitely the best way to try and determine what needs to be done.

**Reply by CaptainOnly** [reply]
Lovely. I'm hoping to get to this in the next week or so.
